lightning/impl/tests/test_primal_cd.py

```diff
@@ -172,7 +174,10 @@ def test_debiasing_l1l2():
                            max_iter=20, C=0.01, random_state=0)
         clf.fit(mult_csc, mult_target)
         assert clf.score(mult_csc, mult_target) > 0.75
-        assert clf.n_nonzero(percentage=True) == 0.08
+        if system() == "Windows":
+            assert 0.07 <= clf.n_nonzero(percentage=True) <= 0.1
```

Comment:
I would be fine with `clf.n_nonzero(percentage=True) <= 0.1` without the platform-specific test.

Reply:
Thanks for your comment! Addressed in 4942a8449b0b1f0879b2a3942f7809f106d70b39. I also added lower bound into assertion to check that number is non-negative, if you don't mind.